dev/ansible/20_upload_files.yml

```diff
@@ -44,8 +44,17 @@
     - example-upload
     - test
 
+- name: Register access key
+  shell: docker exec $(docker ps -a -q --filter ancestor="minio/minio:latest") sh -c "env | grep MINIO_ACCESS_KEY=" | cut -f2 -d"="
+  register: access_key
+
+- name: Register secret key
+  shell: docker exec $(docker ps -a -q --filter ancestor="minio/minio:latest") sh -c "env | grep MINIO_SECRET_KEY=" | cut -f2 -d"="
+  register: secret_key
+  tags: test
```

Comment:
Why are you doing this instead of getting it from vault?

And am I understanding correctly you execute the grep for access_key and secret_key inside the running minio container?

Reply:
Yes. Simple, it should work for all examples (vault-provided-creds and user-provided-creds)

Reply:
Oh, obviously, that makes sense. Thanks :D